src/dispatcher.js

```diff
@@ -165,17 +166,17 @@ export default class Dispatcher {
 
   dispatchSwitchEvent (event, element, direction) {
     if (event.altKey || event.ctrlKey || event.metaKey || event.shiftKey) return
-
     const cursor = this.selectionWatcher.getSelection()
     if (!cursor || cursor.isSelection) return
 
-    if (direction === 'up' && cursor.isAtFirstLine()) {
+    const totalCharCount = getTotalCharCount(element)
+    if (direction === 'up' && (cursor.isAtFirstLine() || totalCharCount === 0)) {
```

Comment:
Do you know why `isAtFirstLine()` does not work here?

Reply:
What do you mean by "does not work here"?